collectors/carbonblack/test/carbonblack_mock.js

```diff
@@ -19,6 +19,8 @@ process.env.paws_api_secret = "api-secret";
 process.env.collector_streams = "[\"AuditLogEvents\", \"SearchAlerts\",\"SearchAlertsCBAnalytics\", \"SearchAlertsWatchlist\"]";
 process.env.paws_collector_param_string_2 = "carbonblackOrgKey";
 process.env.paws_endpoint = "https://api-url.conferdeploy.net";
+process.env.collector_streams_null = "[\"AuditLogEventsCB\", \"SearchAlertsCB\",\"SearchAlertsCBAnalyticsCB\", \"SearchAlertsWatchlistCB\"]";
```

Comment:
collector_streams_null what is the purpose of adding this env variable?

Reply:
<img width="683" alt="image" src="https://user-images.githubusercontent.com/98916725/219317112-356996bd-aead-4ed1-bc2a-41b6d96a2706.png">

Reply:
rename the variable name collector_streams_null